build.sbt

```diff
@@ -39,6 +39,13 @@ def buildLevelSettings: Seq[Setting[_]] =
 
 def commonSettings: Seq[Setting[_]] =
   Seq[SettingsDefinition](
+    headerLicense := Some(HeaderLicense.Custom(
+      """|sbt
+         |Copyright 2011 - 2017, Lightbend, Inc.
+         |Copyright 2008 - 2010, Mark Harrah
+         |Licensed under BSD-style license (see LICENSE)
```

Comment:
I suggest we use [SPDX id](https://spdx.org/licenses/) and say:

> Licensed under BSD-3-Clause license (see LICENSE)

since there are many variants of BSD-style.